src/cli/cli.ts

```diff
@@ -18,10 +18,10 @@
 
 /* eslint-disable no-console */
 
-import * as path from 'path';
+import path from 'path';
 import * as program from 'commander';
 import * as os from 'os';
```

Comment:
sometimes you missed os/util/ws was this intended?

Reply:
I just fixed all the errors.

Reply:
I did a find/replace for some common ones so I might have been over eager.